revolv/static/css/screen.css

```diff
@@ -8380,6 +8620,145 @@ header.edit-project-header > .container > h1 {
   #modal-how-it-works .info-row .step-area {
     display: none;
   }
+
+
+/* 30-Jun */
```

Comment:
Same thing with this comment. Please update it to describe what the styles below apply to.